src/serialization/portable/PortableSerializer.ts

```diff
@@ -73,7 +69,7 @@ export class PortableSerializer implements Serializer {
                 input.position(backupPos);
             }
         }
-        let reader: PortableReader;
+        let reader: DefaultPortableReader;
```

Comment:
Why change to `DefaultPortableReader` from `PortableReadier?

Reply:
`PortableReader` doesn't have `end` function so I needed to use `DefaultPortableReader`.

Reply:
👍